master/internal/rbac/audit/audit.go

```diff
@@ -0,0 +1,34 @@
+package audit
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// GetMethod is the string for the get method.
```

Comment:
not sure where we use this but another option would be to use ```http.MethodGet``` constants and so on?

Reply:
ah good catch - this is vestigial from the previous version